src/sidebar/components/ShareDialog/ShareDialog.tsx

```diff
@@ -9,23 +9,34 @@ import ShareAnnotations from './ShareAnnotations';
 import TabHeader from './TabHeader';
 import TabPanel from './TabPanel';
 
+export type ShareDialogProps = {
+  /** If true, the share tab will be rendered. Defaults to false */
+  shareTab?: boolean;
+  /** If true, the export tab will be rendered. Defaults to false */
+  exportTab?: boolean;
+  /** If true, the import tab will be rendered. Defaults to false */
+  importTab?: boolean;
+};
+
 /**
  * Panel with sharing options.
- * - If export feature flag is enabled, will show a tabbed interface with
- *   share and export tabs
+ * - If provided tabs include `export` or `import`, will show a tabbed interface
  * - Else, shows a single "Share annotations" interface
  */
-export default function ShareDialog() {
+export default function ShareDialog({
+  shareTab,
+  exportTab,
+  importTab,
+}: ShareDialogProps) {
   const store = useSidebarStore();
   const focusedGroup = store.focusedGroup();
   const groupName = (focusedGroup && focusedGroup.name) || '...';
   const panelTitle = `Share Annotations in ${groupName}`;
 
-  const showExportTab = store.isFeatureEnabled('export_annotations');
-  const showImportTab = store.isFeatureEnabled('import_annotations');
-  const tabbedDialog = showExportTab || showImportTab;
+  const tabbedDialog = exportTab || importTab;
```

Comment:
It is possible that we have a state where there is only the export tab or only the import tab, but this is expected to be a temporary state until we complete the feature. I think it would be fine to have a tabbed dialog with a single tab in that case.

Reply:
Yep. I decided not to handle that because it's not a "real world" possible combination, and also not very critical if it happens by accident.